### ISSUE TITLE
try this model on my data with pre-trained model by market1501

### ISSUE THREAD

**Issue by kazuki-can** [lannguyen0910/deep-efficient-person-reid#7]
Hi, I am new to this model.
Thank you for this great work. This is quite interesting.
I have a question about evaluation on my data.
How can I use this pre-trained model?
I want to try with weights by market1501.
Can I write like below?
`       gallery_feats = torch.load(PATH_TO_MY_GALLERY_DATA)
        img_path = np.load(os.path.join(config.log_dir, config.img_path))
        query_img = query_image
        input = torch.unsqueeze(transform(query_img), 0)
        input = input.to(config.device)
        with torch.no_grad():
            query_feat = model(input)

        dist_mat = cosine_similarity(query_feat, gallery_feats)
        indices = np.argsort(dist_mat, axis=1)
        visualizer(test_img, camid='mixed', top_k=10,img_size=config.image_size)`
Thank you.
